src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
+  let total = 0;
+
+  if (days >= 3 && days < 7) {
+    total = sum - 20;
+  } else if (days >= 7) {
+    total = sum - 50;
+  } else {
+    total = sum;
```

Comment:
what a magic numbers 🪄
lets create variables for discount amount and days and then use them
it is better for managing them from one place and for understanding what do they mean

Reply:
and avoid such nesting `else if` conditions
This code is hard to read and maintain